src/meta_schedule/database/json_database.cc

```diff
@@ -67,6 +68,10 @@ void JSONFileAppendLine(const String& path, const std::string& line) {
 /*! \brief The default database implementation, which mimics two database tables with two files. */
 class JSONDatabaseNode : public DatabaseNode {
  public:
+  explicit JSONDatabaseNode(String mod_eq_name = "structural")
+      : DatabaseNode(mod_eq_name),
+        workloads2idx_(/*bucket_count*/ 0, WorkloadHash(), WorkloadEqual(GetModuleEquality())) {}
```

Comment:
Since `WorkloadEqual` needs to be aware of the module hashing / equality testing being used, we need to construct this `unordered_map` after the `ModuleEquality` instance is constructed in `DatabaseNode` by L72.